pages/news/2020-06-07-mixxx-2-3-beta-released.html

```diff
@@ -0,0 +1,88 @@
+title: Mixxx 2.3 beta released
+author: Be.
+date: 2020-05-28 01:00:00
```

Comment:
Can you update the date both here and in the filename before we merge this?

Reply:
Whoops, good catch. Fixed.